apps/meteor/app/apps/server/communication/rest.js

```diff
@@ -794,6 +796,39 @@ export class AppsRestApi {
 			},
 		);
 
+		this.api.addRoute(
+			'notify-admins',
+			{ authRequired: true },
+			{
+				async post() {
+					const { appId, appName, message } = this.bodyParams;
+					const workspaceUrl = settings.get('Site_Url');
+					const learnMore = `${workspaceUrl}marketplace/explore/info/${appId}`;
```

Comment:
Sometimes the `Site_Url` will end with a slash and sometimes it won't. I highly recommend ensuring it has one and putting it there if it doesn't. However, my information could be wrong, so I'd like input from others :)

Reply:
@graywolf336 we can ensure that using a regex that matches only at the end of the string. What do you think?

It would be something like that:
```javascript

const example = "http://localhost/"
const regex = new RegExp('\\/$', 'gm')
const result = str.replace(regex, subst);

console.log('Result:', result)
```

This way we ensure the site url doesn't have a / at the end of the url and we can add them.

Reply:
Digging more into the code, I've found out that they assume Site_Url always doesn't have the / at the end 🤔 But we may guarantee that through my suggestion above

Reply:
Yeah, I'd rather error on the side of caution than just making an assumption. However, someone more experienced can provide more context and thoughts.